05-regression.Rmd

```diff
@@ -406,7 +417,7 @@ get_regression_table(score_model) %>%
                 latex_options = c("hold_position"))
 ```
 
-Let's first focus on interpreting the regression table output in Table \@ref(tab:regtable), and then we'll later revisit the code that produced it. In the `estimate` column of Table \@ref(tab:regtable) are the intercept $b_0$ = `r evals_line[1]` and the slope $b_1$ = `r evals_line[2]` for `bty_avg`. Thus the equation of the regression line in Figure \@ref(fig:numxplot3) follows:
+Let's first focus on interpreting the regression table output in Table \@ref(tab:regtable), and then we'll later revisit the code that produced it. In the `estimate` column of Table \@ref(tab:regtable) are the intercept $b_0$ = 3.880 and the slope $b_1$ = `r evals_line[2]` for `bty_avg`. Thus the equation of the regression line in Figure \@ref(fig:numxplot3) follows:
```

Comment:
Let's avoid hard-coding here.

Reply:
The issue is that the sig figs don't match what's in the table. I think we'll just have to forgo this for this edition